rebase.sh

```diff
@@ -300,6 +313,8 @@ resolve_conflicts() {
       apply_code_package_changes
     elif [[ "$conflictingFile" == "code/product.json" ]]; then
       apply_code_product_changes
+    elif [[ "$conflictingFile" == "code/extensions/mangle-loader.js" ]]; then
```

Comment:
does the PR contain the corresponding part in the `.rebase` folder?
this PR contains a lot of commits, so maybe I just missed it...

Reply:
> maybe I just missed it...

Good catch! :+1: Thanks a lot for noticing the missing file I forgot to commit :facepalm: 
Added a new replacement rule: https://github.com/che-incubator/che-code/pull/252/commits/396fba65f4f850721017d32a8f85a5c7f64ec85f